pkg/utils/tar.go

```diff
@@ -99,7 +101,7 @@ func untar(r io.Reader, dir, relpath string) (err error) {
 		if err != nil {
 			return err
 		}
-		abs := filepath.Join(dir, rel)
+		abs := pathing.SanitizeFilepath(filepath.Join(dir, rel))
 
 		fi := f.FileInfo()
 		mode := fi.Mode()
```

Comment:
I think the 2 functions `validRelativeDir` and `validRelPath` and the end of this file might also be relevant for Windows support.